### PR TITLE
Imputation initial demo

### PR DESCRIPTION
**Pull request by austinTalbot7241993**
This demonstrates to users how to impute missing data.

### CODE REVIEW

**Comment by akotlar** [comment]
Thanks! Will take a look in the morning

File: python/python/bystro/examples/Demo_Imputing_Missing_Data.ipynb [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+   "outputs": [],
+   "source": [
+    "\n",
+    "# Import necessary libraries\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from sklearn.metrics import mean_squared_error\n",
+    "\n",
+    "import sys\n",
+    "sys.path.append('/home/austin/Forks/bystro/python/python/bystro/imputation/fancyimpute/')\n",
```
Comment (akotlar) [inline python/python/bystro/examples/Demo_Imputing_Missing_Data.ipynb:27]: Please remove hardcoded paths. For files that you want available to users, we can add them to our s3 bucket, just send them to me, and I will give you the path.

fancy impute doesn't seem to be used? (from soft_impute import SoftImpute). Did you mean from fancy impute import SoftImpute?

**Review by akotlar — CHANGES_REQUESTED** [review]
Remove hardcoded paths

File: python/python/bystro/examples/Demo_Imputing_Missing_Data.ipynb (hunk 1 above)
Comment (austinTalbot7241993) [reply python/python/bystro/examples/Demo_Imputing_Missing_Data.ipynb:27]: We ate fancyimpute for some of our methods which we have as a subdirectory.

**Review by akotlar — APPROVED** [review]
LGTM

**Pull request merged**